src/services/__tests__/boegivar.test.js

```diff
@@ -0,0 +1,11 @@
+import expect from 'expect';
+import { boegivar } from '../boegivar';
+
+describe('boegivar', () => {
+  describe('sayBoegivar', () => {
+    it('returns a message containing parameter firstName', () => {
```

Comment:
at this level there's no need to test for that, it will simply not print anything, however it might be interesting to test that one level up, because if for some reason the structure of `message`.